File: WordPress/Classes/ViewRelated/Post/PostMetaButton.m
#import "PostMetaButton.h"

@implementation PostMetaButton

- (CGSize)intrinsicContentSize
{
    CGSize newSize = [super intrinsicContentSize];
    newSize.width += (self.imageEdgeInsets.left + self.imageEdgeInsets.right);
    newSize.width += (self.titleEdgeInsets.left + self.titleEdgeInsets.right);
    return newSize;
}

@end

File: WordPress/Classes/ViewRelated/Post/PostMetaButton.swift
import UIKit

// Temporary converted to Swift silence the compiler warnings that are treated
// as errors in Objective-C files.
final class PostMetaButton: UIButton {
    override var intrinsicContentSize: CGSize {
        var newSize = super.intrinsicContentSize
        newSize.width += (imageEdgeInsets.left + imageEdgeInsets.right)
        newSize.width += (titleEdgeInsets.left + titleEdgeInsets.right)
        return newSize
    }
}
